summarizer/summarize.js

```diff
@@ -0,0 +1,8 @@
+const { SummarizerManager } = require('node-summarizer');
+
+module.exports = async function (text, numSentences = 3) {
+  const Summarizer = new SummarizerManager(text, numSentences);
```

Comment:
`const summarizer = ...`

Use lower case to start a variable name, only use a capital for a Constructor function name.